CODEOWNERS

```diff
@@ -1,10 +1,10 @@
-/compute_tools/ @neondatabase/control-plane @neondatabase/compute
-/control_plane/ @neondatabase/compute @neondatabase/storage
-/libs/pageserver_api/ @neondatabase/compute @neondatabase/storage
+/compute_tools/ @neondatabase/compute
```

Comment:
I'd also leave the control plane team here. They're contributing quite often

Reply:
Pushed a commit to add the control plane team back.  I think we should try and move in the direction of having one defined owner, but having two here shouldn't be a practical problem if we're okay with PRs to this path requiring double approval.